client/app/bundles/App/Components/Core/Form/Index.jsx

```diff
@@ -1,122 +1,18 @@
-    </div>
-    <div>
-      <Button text="Fieldset" variant="link" onClick={() => { scrollToComponent('#fieldset'); }} />
-    </div>
-    <div>
-      <Button text="Input" variant="link" onClick={() => { scrollToComponent('#input'); }} />
-    </div>
-    <div>
-      <Button text="Control" variant="link" onClick={() => { scrollToComponent('#control'); }} />
-    </div>
-    <div>
-      <Button text="Number Field" variant="link" onClick={() => { scrollToComponent('#number-field'); }} />
-    </div>
-    <div>
-      <Button text="Textarea" variant="link" onClick={() => { scrollToComponent('#textarea'); }} />
-    </div>
-    <div>
-      <Button text="Textarea Field" variant="link" onClick={() => { scrollToComponent('#textarea-field'); }} />
-    </div>
-    <div>
-      <Button text="Text Field" variant="link" onClick={() => { scrollToComponent('#text-field'); }} />
-    </div>
-    <div>
-      <Button text="Select" variant="link" onClick={() => { scrollToComponent('#select'); }} />
-    </div>
-    <div>
-      <Button text="Select Field" variant="link" onClick={() => { scrollToComponent('#select-field'); }} />
-    </div>
-    <div>
-      <Button text="Controlled Input Example" variant="link" onClick={() => { scrollToComponent('#controlled-input-example'); }} />
-    </div>
-    <Markdown id="field-docs" src={FieldDocs} />
-    <Markdown id="props-table-field" src={FieldProps} />
-    <FieldExamples />
-    <Markdown id="fieldset-docs" src={FieldsetDocs} />
-    <Markdown id="props-table-fieldset" src={FieldsetProps} />
-    <FieldsetExamples />
-    <Markdown id="input-docs" src={InputDocs} />
-    <Markdown id="props-table-input" src={InputProps} />
-    <InputExamples />
-    <Markdown id="control-docs" src={ControlDocs} />
-    {/* <Markdown id="props-table-control" src={ControlProps} /> */}
-    <ControlExamples />
-    <Markdown id="number-field-docs" src={NumberFieldDocs} />
-    <Markdown id="props-table-number-field" src={NumberFieldProps} />
-    <NumberFieldExamples />
-    <Markdown id="text-area-docs" src={TextareaDocs} />
-    <Markdown id="props-table-text-area" src={TextareaProps} />
-    <TextareaExamples />
-    <Markdown id="text-area-field-docs" src={TextareaFieldDocs} />
-    <Markdown id="props-table-text-area-field" src={TextareaFieldProps} />
-    <TextareaFieldExamples />
-    <Markdown id="text-field-docs" src={TextFieldDocs} />
-    <Markdown id="props-table-text-field" src={TextFieldProps} />
-    <TextFieldExamples />
-    <Markdown id="select-docs" src={SelectDocs} />
-    <Markdown id="props-table-select" src={SelectProps} />
-    <SelectExamples />
-    <Markdown id="select-field-docs" src={SelectFieldDocs} />
-    <Markdown id="props-table-select-field" src={SelectFieldProps} />
-    <SelectFieldExamples />
-    <Markdown id="controlled-input-example" src={ControlledInputDocs} />
+    <h2>Controlled Form Example</h2>
+    <br />
     <ControlledInput />
   </div>
```

Comment:
We only have access to the lib code from terra-site which is transpiled to ES5 when we consume terra-site package outside of the terra-core repo.

Because of this I decided to avoid adding a code example. Though I think we should look into distributing src code in the terra-site and terra-clincal-site packages so we can start added src code examples to the doc site.